modules/kmm-signing-kmods-in-a-prebuilt-image.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-signing-kmods-in-a-prebuilt-image_{context}"]
+= Signing kmods in a pre-built image
+
+Use this procedure if you have a pre-built image, such as an image either distributed by a hardware vendor or built elsewhere.
+
+The following YAML file adds the public/private key-pair as secrets with the required key names - `key` for the private key, `cert` for the public key. The cluster then pulls down the `unsignedImage` image, opens it, signs the kernel modules listed in `filesToSign`, adds them back, and pushes the resulting image as `containerImage`.
```

Comment:
This is a soft suggestion if you're up for it, but this second sentence has enough actions in it where maybe putting these actions in a list would make for easier reading. But up to you, it's not too bad.

Reply:
No change.